other/DHT_bootstrap.c

```diff
@@ -95,6 +95,44 @@ static void manage_keys(DHT *dht)
     fclose(keys_file);
 }
 
+void print_log(void *context, LOGGER_LEVEL level, const char *file, int line,
```

Comment:
This function is now replicated 3 times in our codebase. Can we make that only 1 and include it in all 3 places?

Reply:
I found only 2 `print_log()`.

1. [other/DHT_bootstrap.c: 98: `void print_log(...)`
](https://github.com/cotox/c-toxcore/blob/4065638097f5aa496fac47fa00364398fd3d6481/other/DHT_bootstrap.c#L98)

2. [other/bootstrap_daemon/src/tox-bootstrapd.c: 178: `void print_log(...)`](https://github.com/cotox/c-toxcore/blob/4065638097f5aa496fac47fa00364398fd3d6481/other/bootstrap_daemon/src/tox-bootstrapd.c#L178)

Where's the 3rd?

Reply:
Found the 3rd, in IRC chat. note it here.

https://github.com/TokTok/c-toxcore/blob/ae7899cab8104fa3c3078a3e61ddfa58a826e39a/auto_tests/helpers.h#L48